tests/test-funcReturn.mb

```diff
@@ -0,0 +1,25 @@
+int function get1(){
+    return 1;
+}
+
+float function getPi(){
+    return 3.14;
+}
+
+bool function getTrue(){
+    return true;
+}
+
+matrix function getMatrix(){
+    matrix a = [1.1, 2.2; 3.3, 4.4];
```

Comment:
@HuaxuanGAO 
Found something interesting. It kept printing this incorrectly 🤔 
```
1.100000 2.200000
3.300000 0.000000
```

Reply:
I tried print it inside the function, the output is correct.
I guess when returning the matrix, its address got corrupted somehow?

```
matrix function getMatrix(){
    matrix a = [1.1, 2.2; 3.3, 4.4];
    printmf(a);
    return a;
}
```